Automattic/jetpack
# Yarn: update yarn.lock to use current dependencies

Fixes a bug where yarn.lock was still referencing an older version of dops-components, and the build was erroring with `ModuleBuildError in plugin 'webpack`

To Test: 
- `rm -rf ~/.yarn-cache`
- `yarn distclean`
- `yarn build`

You should have a successful happy build!
